### ISSUE TITLE
Add remaining endpoints to pyramid server

### ISSUE THREAD

**Issue by thequbit** [mediapublic/mediapublic#37]
We currently have:

```
/users
/user_types
/recording_categories
/organizations
/blogs
```

We are missing:

```
/people
/recordings
/howtos
/playlists
```

I believe these were not implemented by @ryansb during the re-org due to a lack of understanding of what my initial "vision".

**Comment by ryansb** [comment]
That'd be correct. We can powwow at a post-flock event sometime this week about the data model & REST endpoints.

**Comment by thequbit** [comment]
I think we can make that happen!

**Comment by GabeIsman** [comment]
Closed by @thequbit in #77.
